### DISCUSSION TITLE
laravel10中如何使用EasyWeChat 7.2

### DISCUSSION THREAD

**Discussion by xxika** [w7corp/easywechat#2717] [Q&A]
larvel版本10
EasyWeChat 7.2
通过https://github.com/overtrue/laravel-wechat的composer require overtrue/laravel-wechat:^7.2安装laravel版的EasyWeChat，按照这个地址https://easywechat.vercel.app/6.x/client.html方式去使用laravel去调用EasyWeChat的企业微信API接口，无法调用Application的工厂类。

**Comment by overtrue** [comment]
7.2可能得后年吧我猜，现在只到6.X
